src/main/java/rpgcharacters/commands/CommandCreateUser.java

```diff
@@ -30,6 +30,19 @@ public void run(Connection conn) {
 
             Statement stmt = conn.createStatement();
             stmt.execute(query);
+
+            query = "GRANT dbUser "
+                  + "TO (SELECT username "
+                  +     "FROM user "
+                  +     "WHERE username = '" + this.username + "');";
+            stmt.executeQuery(query);
+
+            query = "GRANT dbAdmin "
+                  + "TO (SELECT COUNT( * ) AS userCount "
+                  +     "FROM user "
+                  +     "WHERE userCount = 1);";
+            stmt.executeQuery(query);
+
```

Comment:
I deleted this file in my PR so this code for granting roles will need to be moved to CreateUserMenu. I merged master into this branch so the SQL is there.